t_x: 50.0
t_y: 50.0
t_z: 50.0
t_vx: 0.0
t_vy: 0.0
t_vz: 0.0
m: 0.01
max_thrust: 1.0
min_tgo: 0.01
xf: 0.0
yf: 0.0
zf: 5.0
delta_rho: 10.0
rho_u: 70.0
g: 10.0
mass: 1.0
max_dist: 250.0
max_height: 100.0
k_gui: 0.07